src/view.js

```diff
@@ -37,6 +38,18 @@ class View {
 
         return button;
     }
+
+    createInput = (object) => {
+        const input = document.createElement('input');
+        input.setAttribute('type', (object.type || 'text'));
+        input.setAttribute('placeholder', object.placeholder);
+        input.setAttribute('id', object.id);
+        input.classList.add(...object.classes);
+        input.setAttribute('value', (object.value || ''));
+        object.required ? input.setAttribute('required', object.required ) : '';
```

Comment:
Please delete object.required and write 'required'
attribute require not equal to true!